dash/App/Tokens/AddToken/index.js

```diff
@@ -95,75 +141,81 @@ class AddTokenAddressScreenComponent extends Component {
 
   submit(address) {
     const { chainId } = this.props
-
-    if (this.isConnectedChain()) {
-      this.props.updateTokenData(address, chainId)
-    }
-
-    link.send('tray:action', 'navDash', {
-      view: 'tokens',
-      data: {
-        notify: 'addToken',
-        notifyData: {
-          address,
-          chainId
-        }
-      }
-    })
+    if (!isValidAddress(address))
+      return navForward({
+        error: invalidFormatError,
+        tokenData: undefined,
```

Comment:
do we need to pass undefined here? looks like somewhere else we're making this call without any token data at all